### PR TITLE
Implemented flexbox layout and CSS

### PR DESCRIPTION
**Pull request by Playtests**
Added Category, Rank and Author
Other minor edits, CSS and image optimizations
Draft template for database implementation

### CODE REVIEW

File: resources/public/css/style.css [hunk 1]
```diff
@@ -11,6 +11,8 @@ body {
   font-family: 'Quicksand', sans-serif;
   font-size: 1.2em;
   color: #494586;
+  background-color: #EEFAFC;
+  background-blend-mode: multiply;
```
Comment (pguillebert) [inline resources/public/css/style.css]: what is the purpose of this line

File: resources/templates/about.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
 		    <header>
           <img class="logo" src="img/logo.png" alt="Logo representing a person of unknown gender"/>
 			    <h1>Inclusiv.me</h1>
-  			  <h2>Small actions to  <span class="highlight">change</span> the world</h2>
+  			  <h2>Small actions to <em>change</em> the world</h2>
```
Comment (pguillebert) [inline resources/templates/about.html]: good work !

File: resources/public/css/style.css (hunk 1 above)
Comment (Playtests) [reply resources/public/css/style.css]: Thanks bro, good catch

File: resources/templates/about.html (hunk 2 above)
Comment (Playtests) [reply resources/templates/about.html]: Ty

**Comment by Playtests** [comment]
It has been done my good lad, quick before you change your mind :P

**Pull request merged**